server/game/cards/02.2-DD/LeonardoLeonCavallo.js

```diff
@@ -0,0 +1,40 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class LeonardoLeonCavallo extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Noon: Leonardo Cavallo',
+            playType: ['noon'],
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Choose a dude',
+                choosingPlayer: 'current',
+                cardCondition: {
+                    location: 'play area',
+                    controller: 'any',
+                    condition: card =>
+                        card.gamelocation === this.gamelocation ||
+                        card.isAdjacent(this.gamelocation)

```

Comment:
There is a function specifically for this that uses new word (will be in new WWE format): nearby

```suggestion
                        card.isNearby(this.gamelocation)
```